index.bs

```diff
@@ -12521,6 +12518,19 @@ values passed to \[[Call]]:
 </ol>
 
 
+<h4 id="preventextensions">Platform object \[[PreventExtensions]] method</h4>
+
+The internal \[[PreventExtensions]] method of every [=platform object=] |O|
+that implements an [=interface=] which
+[=support indexed properties|supports indexed=] or
+[=support named properties|named properties=]
+must behave as follows when called:
+
+<ol class="algorithm">
```

Comment:
Oh, dang, I'm not in favor of that personally. But at least it'd be good to be consistent between this and named properties object in the same PR.